src/gosling-track/gosling-track.ts

```diff
@@ -89,47 +90,77 @@ function initProcessedTileInfo(): ProcessedTileInfo {
     return { goslingModels: [], tabularData: [], skipRendering: false };
 }
 
-/**
- * Each GoslingTrack draws either a track of multiple tracks with SAME DATA that are overlaid.
- * @param HGC
- * @param args
- * @returns
- */
-function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
-    if (!new.target) {
-        throw new Error('Uncaught TypeError: Class constructor cannot be invoked without "new"');
+const config: TrackConfig<GoslingTrackOptions> = {
+    type: 'gosling-track',
+    datatype: ['multivec', 'epilogos'],
+    orientation: '1d-horizontal',
+    // @ts-expect-error missing default spec
+    defaultOptions: {
+        // TODO: Are any of these used?
+        // labelPosition: 'none',
+        // labelColor: 'black',
+        // labelTextOpacity: 0.4,
+        // trackBorderWidth: 0,
+        // trackBorderColor: 'black',
+        // backgroundColor: 'white',
+        // barBorder: false,
+        // sortLargestOnTop: true,
+        // axisPositionHorizontal: 'left',
```

Comment:
These don't seem to have any effect?

Reply:
Yes, let's remove the commented-out properties.